CHANGES_NEXT_RELEASE

```diff
@@ -0,0 +1,5 @@
+- Align tests with iota-node-lib (v3.0) refactor
+- Removed plugins to align with ioagente-node-lib (v3.0) refactor
+- Add support to run tests with node 18
+- Set Nodejs 16 as minimum version in packages.json (effectively removing Nodev14 from supported versions)
+- fix for the binary data representation when sending data through MQTT and HTTP (#690)
```

Comment:
This content seems to be old. The content of this file should be:

* Existing content in CHANGES_NEXT_RELEASE at this moment (can be check at https://github.com/telefonicaid/iotagent-json/blob/master/CHANGES_NEXT_RELEASE)
* The entries corresponding to the fixes in this PR

Reply:
I have updated in the CHANGES_NEXT_RELEASE.